cfme/configure/access_control.py

```diff
@@ -1391,7 +1374,6 @@ def update(self, updates):
                 updates.get('name', self.name))
         view = self.create_view(DetailsTenantView, override=updates)
         view.flash.assert_message(flash_message)
-        assert view.is_displayed
```

Comment:
**QUESTION**: Why no longer asserting htat you are on correct place? Problematic UI? Maybe `is_displayed` needs checking?

Reply:
I removed it thinking that its already asserting the flash message, so `view.is_displayed` won't be necessary. Is it still required @mfalesni ?